src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -373,10 +373,9 @@ <h3>[[localize('settings-metrics-header')]]</h3>
         localize: {type: Function, readonly: true},
         shouldShowExperiments: {type: Boolean, computed: "_computeShouldShowExperiments(supportsAccessKeyDataLimit)"},
       },
-      update: function(name, metricsEnabled) {
+      update: function(name) {
```

Comment:
What does this method do? Could you explain and rename it?
Also, you need to update the caller: outline-server-view.html